### PR TITLE
en: add quick-start-with-dm.md

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

- add en/quick-start-with-dm.md
- update zh/quick-start-with-dm.md format
- remove en/get-started.md
- update links in TOC.md and _index.md

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/80
- Other reference link(s):

### CODE REVIEW

File: en/quick-start-with-dm.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+{{< copyable "shell-regular" >}}
+
+```bash
+dmctl --master-addr=127.0.0.1:8261 list-member
+```
+
+You need to check the following information in the returned result:
+
+- Whether there is a `leader` item;
+- Whether the `master` and the `worker` items include all topology information about DM-master and DM-worker.
```
Comment (TomShawn) [inline en/quick-start-with-dm.md:200]:
```suggestion
- Whether the `master` and the `worker` items include all topology information of DM-master and DM-worker.
```

File: en/quick-start-with-dm.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
+---
+title: Quick Start Guide
+summary: Learn how to quickly deploy a DM cluster using binary packages.
+category: how-to
+aliases: ['/docs/tidb-data-migration/dev/get-started/']
+---
+
+# Quick Start Guide
+
+This document describes how to quickly deploy a [TiDB Data Migration](https://github.com/pingcap/dm) (DM) cluster using binary packages.
```
Comment (TomShawn) [inline en/quick-start-with-dm.md:10]: @lichunzhu Please help confirm whether it should be a `DM cluster` or a `DM instance`, thanks!

Comment (csuzhangxc) [reply en/quick-start-with-dm.md:10]: `DM cluster` is better.

Comment (TomShawn) [reply en/quick-start-with-dm.md:10]: @csuzhangxc Thanks!

Comment (lichunzhu) [reply en/quick-start-with-dm.md:10]: Yes. `DM cluster` is better

**Comment by toutdesuite** [comment]
I'll remove `get-started.md` in another PR to avoid complex conflicts
PTAL again @TomShawn @lichunzhu Many thanks

File: en/quick-start-with-dm.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+| DM-worker2 | 127.0.0.1 | 8263 |
+| DM-worker3 | 127.0.0.1 | 8264 |
+
+## Download binary
+
+Before deploying DM, you need to download the binary package, run the upstream and downstream databases, and prepare the data.
+
+### Download DM binary package
+
+Download the latest (DM v2.0) version of the binary package or compile the package manually.
```
Comment (lichunzhu) [inline en/quick-start-with-dm.md:34]:
```suggestion
Download DM v2.0 binary package or compile the package manually.
```
which one is better?

File: en/quick-start-with-dm.md [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+DM_PATH=`pwd` && export PATH=$PATH:$DM_PATH/bin
+```
+
+### Run upstream MySQL
+
+Run two MySQL client services. To start MySQL services using Docker, execute the following commands:
```
Comment (lichunzhu) [inline en/quick-start-with-dm.md:66]:
```suggestion
Run two MySQL services. To start MySQL services using Docker, execute the following commands:
```
It should be mysql server here. The Chinese version is wrong here.

File: en/quick-start-with-dm.md (hunk 3 above)
Comment (toutdesuite) [reply en/quick-start-with-dm.md]: `Download DM v2.0 binary package` is concise. Updated in df4e24e654b45fb98c8f9e2b0b6b231cdc4e06de

File: en/quick-start-with-dm.md (hunk 4 above)
Comment (lichunzhu) [reply en/quick-start-with-dm.md]: Counld you please revise `运行两个 MySQL 客户端的服务，使用 Docker 启动 MySQL，命令如下：` to `运行两个 MySQL 服务，使用 Docker 启动 MySQL，命令如下：` in `zh/quick-start-with-dm.md`?

Comment (toutdesuite) [reply en/quick-start-with-dm.md]: ok

File: en/quick-start-with-dm.md [hunk 5]
```diff
@@ -0,0 +1,283 @@
+---
+title: Quick Start Guide for TiDB Data Migration
+summary: Learn how to quickly deploy a DM cluster using binary packages.
+category: how-to
+aliases: ['/docs/tidb-data-migration/dev/get-started/']
```
Comment (TomShawn) [inline en/quick-start-with-dm.md:5]:
```suggestion
aliases: ['/docs/tidb-data-migration/dev/get-started/','/tidb-data-migration/dev/get-started']
```

**Pull request merged**